src/Desktop/UIAutomation/EventHandlers/TextEditTextChangedEventListener.cs

```diff
@@ -45,13 +46,16 @@ public void HandleTextEditTextChangedEvent(IUIAutomationElement sender, TextEdit
                 {
                     new KeyValuePair<string, dynamic>("TextEditChangeType", type.ToString()),
                 };
-                for (int i = 0; i < array.Length; i++)
+
+                if (array != null)
                 {
-                    m.Properties.Add(new KeyValuePair<string, dynamic>(Invariant($"[{i}]"), array.GetValue(i)));
-                };
+                    for (int i = 0; i < array.Length; i++)
+                        m.Properties.Add(new KeyValuePair<string, dynamic>(Invariant($"[{i}]"), array.GetValue(i)));
```

Comment:
Personally, I'd make line 53 a block with curly braces. Failing to do so makes the next line jump 2 indents, which is awkward

Reply:
The best way to fix this problem is probably to make the code with the `for` loop its own function. I hesitate to add the braces because multiple sets of braces become confusing for us screen reader users (those who don't use Braille displays anyway).